desktop-app/src/renderer/store/features/device-manager/index.ts

```diff
@@ -70,6 +70,12 @@ export const deviceManagerSlice = createSlice({
       state.activeSuite = action.payload.id;
       window.electron.store.set('deviceManager.previewSuites', suites);
     },
+    addSuites(state, action: PayloadAction<PreviewSuite[]>) {
+      window.electron.store.set('deviceManager.previewSuites', []);
+      state.suites = action.payload;
+      state.activeSuite = action.payload[0].id;
+      window.electron.store.set('deviceManager.previewSuites', action.payload);
```

Comment:
This looks like it will override the existing suites as well, wouldn't that result in a loss of data if the user already had any custom suites?

Reply:
sure, should we add also a button to reset the whole thing so they can go to the initial clean state as well?